docs/standard/commandline/syntax.md

```diff
@@ -0,0 +1,572 @@
+---
+title: Command-line syntax overview for System.CommandLine
+description: "An introduction to the command-line syntax that the System.CommandLine library recognizes by default. Mentions exceptions where syntax in the .NET CLI differs. Provides guidance for designing a command-line interface."
+ms.date: 02/11/2022
+no-loc: [System.CommandLine]
+helpviewer_keywords:
+  - "command line interface"
+  - "command line"
+  - "System.CommandLine"
+ms.topic: conceptual
+---
+
+# Command-line syntax overview for System.CommandLine
+
+This article explains the command-line syntax that `System.CommandLine` recognizes. The information will be useful to users as well as developers of .NET command-line apps, including the [.NET CLI](../../core/tools/index.md).
+
+## Tokens
+
+`System.CommandLine` parses command-line input into *tokens*, which are strings delimited by spaces. For example, consider the following command line:
+
+```dotnetcli
+dotnet tool install dotnet-suggest --global --verbosity quiet
+```
+
+This input is parsed into tokens `tool`, `install`, `dotnet-suggest`, `--global`, `--verbosity`, and `quiet`.
+
+Tokens are interpreted as commands, options, or arguments. The command-line app that is being invoked determines how the tokens after the first one are interpreted. The first token identifies the app. The following table shows how `System.CommandLine` interprets the preceding example:
+
+| Token            | Parsed as                         |
+|------------------|-----------------------------------|
+| `dotnet`         | Root command - runs *dotnet.exe*  |
+| `tool`           | Subcommand                        |
+| `install`        | Subcommand                        |
+| `dotnet-suggest` | Argument for install command      |
+| `--global`       | Option for install command        |
+| `--verbosity`    | Option for install command        |
+| `quiet`          | Argument for `--verbosity` option |
+
+A token can contains spaces if it's enclosed in quotation marks (`"`). Here's an example:
+
+```console
+dotnet tool search "ef migrations add"
+```
+
+## Commands
```

Comment:
Commands are often commonly referred to as "verbs". Mentioning that might help some people understand the concept.

Reply:
I mention that under the subcommand section -- should the root command also be called a verb?

Reply:
No, I think you have it right to just associate that term with subcommands.